### DISCUSSION TITLE
Need help - Custom Filter Rules doesnt work

### DISCUSSION THREAD

**Discussion by yodaphone** [AdguardTeam/AdGuardHome#3677] [Q&A]
I use AGH with opnsense and i have this rule to allow all requests but it still doesnt work. If i change my DNS to cloudflare/google it works just fine. When i try to access amazon & there's a redirect thro aax-us-east.amazon-adsystem.com it fails and its not reported in the log too. Any idea how to make this work ?

@@||amazon-adsystem.com^

**Comment by ainar-g** [comment]
Hello, please tell us the following:

1.  Which version are you running?

1.  Have you tried disabling and reenabling the protection to make sure that caches are purged?

1.  Can you provide an example of a query that fails like that?  E.g. a `dig` or an `nslookup` command?

**Reply by yodaphone** [reply]
thank you for the prompt reply

1. Version: v0.107.0-b.11
2. Yes, also purged cache on my PC & browser as well
3. I dont know how to do that

**Reply by yodaphone** [reply]
Ok, finally solved it by changing the Custom Filter Rules to this, not sure if this is event the right syntax but works ;-) 
@@/amazon-adsystem.*/$important
